mmhuman3d/core/visualization/renderer/torch3d_renderer/smpl_renderer.py

```diff
@@ -191,41 +149,29 @@ def forward(
                 2). If render silhouette, the output tensor shape will be
                 (frame, h, w) or (frame, num_people, h, w).
                 3). If render part silhouette, the output tensor shape should
-                be (frame, h, w, n_class) or (frame, num_people, h, w, n_class
-                ). `n_class` is the number of part segments defined by smpl of
-                smplx.
+                be (frame, h, w, 1) or (frame, num_people, h, w, 1
+                ).
         """
-        num_frames, num_person, _, _ = vertices.shape
-        faces = self.raw_faces[None].repeat(num_frames, 1, 1)
+        num_frames = len(meshes)
         if self.frames_folder is not None and images is None:
+
             images = images_to_array(
                 self.frames_folder,
                 resolution=self.resolution,
-                img_format=self.img_format,
+                img_format=self.read_img_format,
                 start=indexes[0],
                 end=indexes[-1],
```

Comment:
`end=indexes[-1]+1`
As the last index is included, end argument should be above it.

Reply:
this is good